### ISSUE TITLE
Culinary Water Suppliers update

### ISSUE THREAD

**Issue by mhilderman** [agrc/gis.utah.gov#1107]
_Please be comfortable to submit an issue if you have a question, want to ask to post new content or if the website isn't working properly for you._

## Issue description
Downloaded retail culinary water suppliers dataset and noticed some areas were not represented accurately within our municipal boundary. 

#### What page is the problem visible
https://gis.utah.gov/data/utilities/retail-culinary-water-suppliers/

#### Expected result
Revision of existing data

#### Actual result
Please see attached .pdf map. I can also distribute an zipped file of the sewer service boundaries within Midvale City and adjacent communities.
[Water Service Boundaries.pdf](https://github.com/agrc/gis.utah.gov/files/3213305/Water.Service.Boundaries.pdf)

_You can drag and drop or paste screen shots from your clipboard to here. Please do it's very helpful!_

**Comment by steveoh** [comment]
@mhilderman thank you for bringing this to our attention!!!

@ZachBeck and @mheagin can you work with @mhilderman to fix this data issue please? Please keep this issue notified of the outcome etc.

Thanks!

**Comment by mheagin** [comment]
a quick look shows
UTILITIES.RetailCulinaryWaterServiceAreas
Steward(s): Utah Division of Water Resources, Utah Department of Natural Resources 
last update 12/17/2015
Updated "As needed"
Data Source - Water Resources
Contact - Adam Clark - asellars@utah.gov
Endpoint - https://maps.dnr.utah.gov/arcgis/rest/services/WRE/RetailCulinaryWaterServiceAreas/MapServer		Seems to be dead link.

no updates since end of 2015, not all of those districts are in the tax entities (use fees not taxes), I think they have to register through Water Resources.
I will talk to Zach and see what he know's

**Comment by mheagin** [comment]
Looks like it is water resources or DEQ that has access to all those districts. 
We'll contact Adam and see who is or will be updating that feature class

**Comment by mheagin** [comment]
Heard back from Adam Clark,  adamclark@utah.gov
"Hi Mike,
  Yes, I am the data steward over the retail culinary water service areas.  As a division, we are pulling away from using and maintaining SDE feature classes, so the culinary service areas feature class hasn't been updated in awhile.  In fact, we will probably remove that feature class pretty soon from our SDE.  Instead, we are pushing forward with using ArcGIS Online and Open Data.  We have a feature service in AGOL that will be updated weekly that can be used as the most up to date layer.  I'm still working on the script for the automation of that.
  I'm not sure if you guys can easily pull that feature service for use in the SGID or not.  We are also considering changing the name of the feature service to Culinary Water Service Areas, because the geodatabase now contains non-public, self-supplied industry, and public non-community service areas.  The original feature service only had retail public community systems.
  So, with that information in mind, what is the best way going forward for you guys to get that data from us?  Also, we will soon be adding new feature classes of secondary water service areas and irrigation company service areas to AGOL.  Those feature services can also be added to the SGID.
Regards,
Adam Clark, Lead GIS Analyst, Utah Division of Water Resources"
Looks like we can turn this over to the SGID Data Distribution Team, let them decide how we are going to deal with the data now.

**Comment by steveoh** [comment]
@mhilderman how does http://opendata.gis.utah.gov/datasets/utahDNR::retail-culinary-water-service-areas look?

**Comment by mheagin** [comment]
1. Talked to Greg he will discuss it with the SDE group
2. emailed Matt H with update and link to Utah Division of Water Resources data in AGOL and Open Data
3. emailed Adam with update of where AGRC is with his data, and asked if they had a preference on how we shared their data.
You want to close this  issue not ?

**Comment by steveoh** [comment]
@mheagin let's close this when the 3rd item is resolved... Do you think that is reasonable?

**Comment by mheagin** [comment]
Sounds good, I will let you know if I hear anything.

**Comment by steveoh** [comment]
Any updates on this?

**Comment by mheagin** [comment]
Have not heard back from Adam, not sure if Greg or Michael have talked to the UDWR group as part of the SGID rebuild. I will check with Greg.

**Comment by mheagin** [comment]
I checked with Greg and Michael they said UDWR is only going to update the AGOL data and not their data in SDE. So they are going to make that change.  For us we are done.

**Comment by steveoh** [comment]
Are we done or do we need to update the website to point to the agol instead of the out of date download that we have? @ZachBeck is there a process to remove items from back seat driver or what happens in this situation?

**Comment by mheagin** [comment]
I have done what I can do, yes I am done. It is up to Zach and Greg now I guess

**Comment by ZachBeck** [comment]
The website will need to point at their AGOL if that's the route they're going now.
I don't know if Keaton had backseat driver running as a scheduled task for certain feature classes, I have always run it manually.

**Comment by steveoh** [comment]
@stdavis has it set up as a scheduled task. Who is going to make the changes for the agol link? It needs to happen here and the stewardship sheet.

**Comment by ZachBeck** [comment]
I can do it. Did they ever give us a link?

**Comment by mheagin** [comment]
here is what Aaron, couple of links.
"Aaron Austin
Hi
Adam is out in the field a lot this week. I will try to respond and maybe he can correct me later if I've got anything wrong.
Our open data is found at http://dwre-utahdnr.opendata.arcgis.com/ but we typically publicize water.utah.gov/opendata which redirects there.
Currently though I don't think the specific layer these emails have been referring to is available there. That shouldn't be too hard to change.
The link to the hosted feature layer is https://utahdnr.maps.arcgis.com/home/item.html?id=45ea64f5bb974ec9a03990df05eafd48
and the rest service is https://services.arcgis.com/ZzrwjTRez6FJiOq4/arcgis/rest/services/Retail_Culinary_Water_Service_Areas/FeatureServer
I think we also intended to share it here http://opendata.gis.utah.gov
Hopefully these are the kinds of things you were looking for, but if not let me know and I'll keep trying.
Thanks!"
will these work?
